README.md

```diff
@@ -0,0 +1,66 @@
+# Algebra In Lean
+
+This is the repository for Algebra In Lean (AIL), an interactive problem set
+that takes users through an advanced course in Lean through the lens of abstract
+algebra. The goal of this problem set is to introduce users to formalization
+and Lean 4 while also assisting in writing rigorous proofs in
+group theory. Beginning with introductions to logic-based reasoning in Lean, each chapter takes users through various topics in group theory. Akin to Kevin
```

Comment:
"After an introduction to" sounds better since chapter 0 isn't a topic in group theory.